sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ImplementationBridgeHelpers.java

```diff
@@ -1067,7 +1101,6 @@ CosmosBatchRequestOptions setConsistencyLevel(CosmosBatchRequestOptions cosmosBa
             CosmosBatchRequestOptions setHeader(CosmosBatchRequestOptions cosmosItemRequestOptions, String name, String value);
             Map<String, String> getHeader(CosmosBatchRequestOptions cosmosItemRequestOptions);
             List<String> getExcludeRegions(CosmosBatchRequestOptions cosmosBatchRequestOptions);
```

Comment:
`getExcludeRegions` seems unused - can be removed.

Reply:
removed